upcloud/resource_upcloud_server.go

```diff
@@ -299,24 +308,39 @@ func resourceUpCloudServerCreate(ctx context.Context, d *schema.ResourceData, me
 	if err != nil {
 		return diag.FromErr(err)
 	}
-	server, err := client.CreateServer(r)
+	s, err := client.CreateServer(r)
```

Comment:
Is there a reason why we use `s` instead of the more descriptive `server` as before?

Reply:
server is used in here https://github.com/UpCloudLtd/terraform-provider-upcloud/pull/150/files#diff-932d31a4bcd943933a162fe0586b58bd2b829b2e5a3b9f81ab060c35c5cbcdceR322
which conflitcts

Reply:
ahh, right, maybe just `serverdetails` or `createdserver`, then?